src/vm/peimage.cpp

```diff
@@ -969,7 +969,20 @@ PTR_PEImageLayout PEImage::CreateLayoutMapped()
     {
         // For CoreCLR, try to load all files via LoadLibrary first. If LoadLibrary did not work, retry using 
         // regular mapping - but not for native images.
-        pLoadLayout = PEImageLayout::Load(this, TRUE /* bNTSafeLoad */, m_bIsTrustedNativeImage /* bThrowOnError */);
+        pLoadLayout = PEImageLayout::Load(this, FALSE /* bNTSafeLoad */, m_bIsTrustedNativeImage /* bThrowOnError */);
```

Comment:
What happens on Unix when we try to load IJW image? We should make sure to have a good error message for that. (It is ok to open issue on this if you do not want to do this in this PR.)

Reply:
It would be nice not to block that IJW assembly loading since the format and runtime aren't OS-specific, the contents are. I'm not sure whether the failure right now on Unix is the OS rejecting the load or AVing when the module constructor tries to call an import that the OS hasn't fixed up.

Reply:
The format of IJW assemblies is Windows-specific.

We do not call OS to load PE images on non-Windows. We have our own limited PE file loader on Unix that is just enough to handle IL-only images and Unix-specific R2R images.